src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -191,7 +191,7 @@ public void deleteOldJobs() {
   @Override
   public void resendExportedFile(UUID jobId) {
     org.folio.des.domain.dto.Job job = get(jobId);
-    if (CollectionUtils.isEmpty(job.getFiles())) {
+    if (CollectionUtils.isEmpty(job.getFileNames())) {
```

Comment:
For download method I propose to do the same to have consistent validation

Reply:
When we try to download, we check the entity and there should be a link in it.

Reply:
Yes I know, but better I think to do the similar validation in both methods